packages/frontend-2/lib/auth/composables/auth.ts

```diff
@@ -396,6 +397,17 @@ export const useAuthManager = (
       forceFullReload: boolean
     }>
   ) => {
+    if (isSsoEnabled.value) {
+      await fetch(new URL('/api/v1/auth/logout', apiOrigin).toString(), {
```

Comment:
@cdriesler  Y tho? Isn't deleting the token from the client enough? Surely if its enough for our main auth, its also enough for SSO?

Reply:
It's enough to sign you out, but not enough to clear the session. We are not linking "SSO sessions" to the web app sign-in because of how we want them to work with PATs:

- A PAT represents a given user
- A PAT is used in connectors (or other things) to access project data
- An SSO session is created when a user signs in successfully with SSO
- If a the user a PAT is associated with has a valid SSO session for the workspace being accessed, allow
  - If not, disallow, redirect to SSO flow on app

The provision here for clearing the SSO session is to make "sign out" literally mean sign in for SSO access as well.